pkg/api/api.go

```diff
@@ -442,6 +434,23 @@ func (hs *HTTPServer) registerRoutes() {
 		adminRoute.Get("/ldap/status", routing.Wrap(hs.GetLDAPStatus))
 	}, reqGrafanaAdmin)
 
+	// Administering users
```

Comment:
There are a bunch of these endpoints which I missed when going over this, we'll have to make sure to extract the other `admin/users` endpoints from further up in the code as well.

Reply:
Yeah I went through few of them, will double check if I missed anything!